.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+## Motivation
+Add references to relevant tickets or a short description about what motivated you do it. (E.g Trello Card AND/OR GitHub ISSUE: https://github.com/integr8ly/tutorial-web-app/issues) 
+
+## What
+Add an short answer for: What was done in this PR? (E.g Don't allow users has access to the feature X.)
+
+## Why
+Add an short answer for: Why it was done? (E.g The feature X was deprecated.)
+
+## How
+Add an short answer for: How it was done? (E.g By removing this feature from ... OR By removing just the button but not its implementation ... ) 
+
+## Verification Steps
+Add detailed steps required to check this change. Following an example.
+ 
+1. Go to `XX >> YY >> SS`
+2. Create a new item `N` with the info `X`
+3. Try to edit this item 
+4. Check if in the left menu the feature X is not so long present.
+
+## Checklist:
+
+- [ ] Code has been tested locally by PR requester
+- [ ] Changes have been successfully verified by another team member 
```

Comment:
Do we need this?  PR has to be approved by 1 person to have it committed.

Reply:
@dlabaj This should include what steps make sense for the process on PRs on your team and this repo. These are just what we use on my team, it is required that another team member has not only reviewed the code but verified the fix through verification steps either locally or more ideally on a cluster. 

Do ye have a process like this for this web app?

Reply:
okay we can leave this then and see how it goes.